drivers/video/gc2145.c

```diff
@@ -1126,10 +1129,18 @@ static int gc2145_init(const struct device *dev)
 {
 	struct video_format fmt;
 	int ret;
-
-#if DT_INST_NODE_HAS_PROP(0, reset_gpios)
 	const struct gc2145_config *cfg = dev->config;
+	(void) cfg;
+
+#if DT_INST_NODE_HAS_PROP(0, pwdn_gpios)
+	ret = gpio_pin_configure_dt(&cfg->pwdn_gpio, GPIO_OUTPUT_INACTIVE);
+	if (ret) {
+		return ret;
+	}
 
+	k_sleep(K_MSEC(10));
```

Comment:
If you happen to remember why 10ms, a comment would be a nice bonus.

Reply:
It's just an arbitrary large enough delay to make sure the sensor has had enough time settle. We can probably lower this if you like.

Reply:
Thank you, good to know for anyone who troubleshoot power sequence, which can vary a bit from board to board. No issue at all!